### ISSUE TITLE
Can't match cursorSave and cursorRestore

### ISSUE THREAD

**Issue by SamVerschueren** [chalk/ansi-regex#36]
Today I was working with the regex and noticed that [`cursorSave`](https://github.com/sindresorhus/ansi-escapes/blob/main/index.js#L49) and [`cursorRestore` ](https://github.com/sindresorhus/ansi-escapes/blob/main/index.js#L50) don't get stripped with `strip-ansi`.

I didn't check the other use cases yet, but wanted to make sure first that it is correct what I assume here.

It is the case, happy to send a PR.

**Comment by Qix-** [comment]
Yep good catch, these seem to be missing looking at the pattern again. A PR would be awesome :)

**Comment by SamVerschueren** [comment]
I'm wondering, would it make sense to add `ansi-escapes` as dev dependency and then call `ansi-escapes` in the tests so we know for sure that it strips everything that `ansi-escapes` support?

**Comment by Qix-** [comment]
@SamVerschueren I think that makes a lot of sense. :)
